packages/libs/src/sdk/api/users/UsersApi.ts

```diff
@@ -461,6 +463,34 @@ export class UsersApi extends GeneratedUsersApi {
     return await this.claimableTokens.sendTransaction(transaction)
   }
 
+  /**
+   * Submits a reaction to a tip being received.
+   * @hidden
+   */
+  async submitReaction(
```

Comment:
nit: `sendTipReaction`? 

Also (no need to change here) in retrospect I think both this and `sendTip` should probably be in `TipsApi` 🤔

Reply:
I thought about doing that but I went this route since the tip precedent was already set. Also the TipsApi is only reads and I wasn't sure how much more work adding writes would be. Would be a good follow up to this.

Reply:
Actually I think I remember why I did this - I think sending a tip needs to get the user bank so its easier to have in the user api

Reply:
ah makes sense